### ISSUE TITLE
Hyperlinks can be added for improved understand-ability of Basic Image Classification Tutorial 

### ISSUE THREAD

**Issue by worldpeaceaspirer** [tensorflow/tensorflow#47552]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/tutorials/keras/classification#import_the_fashion_mnist_dataset
https://www.tensorflow.org/tutorials/keras/classification#set_up_the_layers
https://www.tensorflow.org/tutorials/keras/classification#compile_the_model
https://www.tensorflow.org/tutorials/keras/classification#feed_the_model

## Description of issue (what needs changing):
1. The phrase, **`load the Fashion MNIST data`** In the statement, **`Import and load the Fashion MNIST data directly from TensorFlow:`**, can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/datasets/fashion_mnist/load_data

2. The word, **`layer`**, In the statement, **`The basic building block of a neural network is the layer`** can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/layers

3. For better readability, The arguments in the Dense Layers can be named i.e., `tf.keras.layers.Dense(128)` can be replaced with `tf.keras.layers.Dense(units = 128)`

4. The word **`compile`** in the statement, **`These are added during the model's compile step:`** can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/Model#compile

5. The word **`Loss Function`** can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/losses

6. The word **`Optimizer`** can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/optimizers

7. The word **`Metrics`** can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/metrics

8. **`model.fit`** in the statement, **`To start training, call the model.fit method—`** can be linked to https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit

### Clear description

Adding the Hyperlinks for each API helps the beginners understand the functionality.

**Comment by ymodak** [comment]
Closing since the associated PR has been merged. Thanks!
